models/project_issue.go

```diff
@@ -200,14 +201,15 @@ func MoveIssueAcrossProjectBoards(issue *Issue, board *ProjectBoard) error {
 	}
 
 	pis.ProjectBoardID = board.ID
-	if _, err := sess.ID(pis.ID).Cols("project_board_id").Update(&pis); err != nil {
+	pis.Sorting = sorting
+	if _, err := sess.ID(pis.ID).Cols("project_board_id").Cols("sorting").Update(&pis); err != nil {
 		return err
 	}
 
 	return sess.Commit()
 }
 
 func (pb *ProjectBoard) removeIssues(e db.Engine) error {
-	_, err := e.Exec("UPDATE `project_issue` SET project_board_id = 0 WHERE project_board_id = ? ", pb.ID)
+	_, err := e.Exec("UPDATE `project_issue` SET project_board_id = 0, sorting = 0 WHERE project_board_id = ? ", pb.ID)
 	return err
```

Comment:
Out of context question: Does this query has a bug? `ProjectBoard` is atm the name for column from what I know #13802. It uses the `project_board_id` which in my understanding is the column number, which can exists in multiple boards. Shouldn't it be: `UPDATE project_issue SET project_board_id = 0, sorting = 0 WHERE project_board_id = ? AND project_id = ?`

Reply:
No problem here. a `project board(column)` belongs to a project. You can try it on https://try.gitea.io/

Reply:
Ahh and it works for the default column as it would just set `project_board_id = 0` again.